specs/protocol/deputy-pause-module.md

```diff
@@ -104,28 +96,27 @@ replaced.
 > typically only rely on a subset of the following assumptions. Different invariants may rely on
 > different assumptions. Refer to individual invariants for their dependencies.
 
-### aSCP-001: Superchain-wide pause authorization process is well-defined
+### aSCP-001: Pause authorization process is well-defined
 
-We assume that the process by which a Superchain-wide pause can be authorized is well-defined such
-that the cases in which the pause should and should not be triggered are apparent.
+We assume that the process by which a pause can be authorized is well-defined such that the cases
+in which the pause should and should not be triggered are apparent.
 
-### aSCP-002: Superchain-wide pause authorization process is robust
+### aSCP-002: Pause authorization process is robust
 
-We assume that the process by which a Superchain-wide pause can be authorized correctly accounts
-for all cases in which such a pause would be necessary.
+We assume that the process by which a pause can be authorized correctly accounts for all cases in
+which such a pause would be necessary.
```

Comment:
This sounds pretty similar to the first assumption, what makes these unique from one another?

Reply:
They're pretty similar. The idea here is that the first assumption is saying "we know all of the circumstances in which the pause should or should not be used" and the second is saying "if such a circumstance occurs, the pause *will* be used".

Reply:
Got it, I'd suggest tweaking the phrasing a bit to make that more clear. Feel free to resolve this thread afterwards